kodilla-exception/src/main/java/com/kodilla/exception/test/SearchEngineRunner.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.exception.test;
+
+public class SearchEngineRunner {
+    public static void main(String[] args) {
+        Flight flightFromWarsaw = new Flight("Warsaw", "Moscow");
+
+        SearchEngine searchEngine = new SearchEngine();
+
+        try {
+            if (searchEngine.findFlight(flightFromWarsaw)) {
+                System.out.println("The flight has been found.");
+            } else {
+                System.out.println("The flight has not been found.");
+            }
+
+        } catch (RouteNotFoundException e) {
+            System.out.println("Please provide the correct name of the airport. " + e);
```

Comment:
Warto wykorzystać również `message` zawarty w wyjątku (tworzony w linii 17 `SearchEngine`, tutaj można go użyć: `e.getMessage()`. Pamiętaj - wyjątek to zwykły obiekt, ma metody i pola.